### DISCUSSION TITLE
WMBC TV - Newton, NJ/New York

### DISCUSSION THREAD

**Discussion by LPhilip1999** [iptv-org/community#809] [Q&A]
### Channel Name

WMBC TV - Newton, NJ/New York

### Channel ID

WMBCDT1.us

### Notes

Please give the M3U8 link for WMBC TV Newton, NJ/New York

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
It doesn't seem that they stream live from their website. If they do get added over MyBTV, I suppose @MapGuy11 can work it out.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19783
